### PR TITLE
KEYCLOAK-18939 Release notes for 15.0.1

### PR DESCRIPTION
**Pull request by mposolda**

### CODE REVIEW

File: release_notes/topics/15_0_1.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+= Highlights
+
+This release contains some important bugfixes. In addition, We would like to Thank You to https://github.com/leandrobortoli[Leandro José de Bortoli] for his contributions to the
```
Comment (andymunro) [inline release_notes/topics/15_0_1.adoc]: bug fixes
Change "Thank You to" to "thank"

File: release_notes/topics/15_0_1.adoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+FAPI related functionalities like JARM support and improvements in CIBA.
```
Comment (andymunro) [inline release_notes/topics/15_0_1.adoc]: Always change "like" to "such as"

**Review by andymunro — CHANGES_REQUESTED** [review]
A few minor changes, Marek.

File: release_notes/topics/15_0_1.adoc (hunk 2 above)
Comment (mposolda) [reply release_notes/topics/15_0_1.adoc]: My favourite mistake :(  Hopefully one day, I will be able to get it right :)

**Comment by mposolda** [comment]
@andymunro Applied all your suggestions. Thanks!

**Pull request merged**